cpp/oneapi/dal/detail/error_messages.cpp

```diff
@@ -182,6 +182,7 @@ MSG(subgraph_isomorphism_is_not_implemented_for_labeled_edges,
     "Subgraph isomorphism is not implemented for labeled edges")
 MSG(non_zero_max_match_count_is_not_supported, "Non-zero max_match_count is not supported")
 MSG(incorrect_index_is_returned, "Internal error: incorrect index is returned")
+MSG(invalid_attributes, "Internal error: invalid attributes")
```

Comment:
What are invalid attributes?

Reply:
used when null pointer is set as array with vertex or edge attributes in kernel code

Reply:
invalid graph attributes pointer